org.eclipse.winery.model.tosca/src/main/java/org/eclipse/winery/model/tosca/TExtensibleElements.java

```diff
@@ -154,6 +154,8 @@ protected void setWPDElement(WinerysPropertiesDefinition res) {
     }
 
     protected void setWPDNamespace(WinerysPropertiesDefinition res) {
+        // bo be overridden by subclasses willing to use getWinerysPropertiesDefinition()
```

Comment:
should this be marked as `abstract` then, to force said overriding? This class is already abstract so we're not giving up anything for that

Reply:
I was not sure if all TExtensibleElements actually need WPD support, since initially only TEntityType supported them. This is an attempt to reuse the method in both TEntityTypes and TServiceTemplates without duplicating it and at the same time without putting additional requirements on TExtensibleElements.